Kentico/training-kentico-k12
# Feature/optimize returned db columns

### Motivation
Change: Optimize Columns Returned from Repositories

- Revised the Repository Queries to use .Columns instead of .AddColumns to optimize database access.
- Using .AddColumns returns at least 150+ columns to the query, .Columns only returns what columns are specified.
- Added NodeSiteId to the columns returned to function during PreviewMode.

### How to test

- Turn on Kentico SQL Debugging
- Turn on Glimpse
- Review the Queries in the Glimpse Kentico SQL tab.  There should be no **select** * in the queries.
